### PR TITLE
Implement consumer portal feature and Web client

### PR DESCRIPTION
**Pull request by tothandras**
- Create web client
- Update quickstart
- Add Next.js dashboard example
image

### CODE REVIEW

File: api/client/web/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# OpenMeter Web Client
+
```
Comment (hekike) [inline api/client/web/README.md:2]: Please add a blurb about when users should use this library. Maybe add a similar to Node SDK too.
Something like:
> Client library for integrating web applications with OpenMeter. Useful to build customer dashboards. The library comes with integrations for React hooks.

File: api/client/web/src/react/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+import { useContext, useMemo } from 'react'
+import { OpenMeterClient } from '../client'
+import { OpenMeterContext } from './context'
+
+export function useOpenMeter() {
+  const context = useContext(OpenMeterContext)
+
+  if (!context) {
+    throw new Error('useOpenMeter must be used within an OpenMeterProvider')
```
Comment (hekike) [inline api/client/web/src/react/hooks.ts:11]: Consider linking to docs [api/client](https://openmeter.io/docs/portal)

File: api/openapi.yaml [hunk 3]
```diff
@@ -481,8 +537,8 @@ components:
       type: object
       required:
         - subject
-        - expiresAt
-        - token
+        # - expiresAt
```
Comment (hekike) [inline api/openapi.yaml:540]: WHy is this commented out?

File: examples/customer-portal-nextjs/.env.example
Comment (hekike) [inline examples/customer-portal-nextjs/.env.example:1]: Wondering if example should be called: `customer-dashboard-nextjs`?

File: api/openapi.yaml (hunk 3 above)
Comment (tothandras) [reply api/openapi.yaml:540]: The OpenAPI generator doesn't support readOnly, writeOnly: https://github.com/drwpow/openapi-typescript/issues/604

**Pull request merged**